src/Configuration/EnvTrait.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Equip\Env;
+
+trait EnvTrait
```

Comment:
Probably need a corresponding PR to drop [this](https://github.com/wheniwork/api-configuration/blob/master/src/EnvTrait.php) and update other files that use it in that repo.